### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by marcusngwenjian**
Update badge and acknowledgement of address book 4
<br>
@CS2103AUG2016-W15-C3/documentation Ready for review

### CODE REVIEW

**Comment by Mamtha3005** [comment]
Documentation Reviewed

**Comment by marcusngwenjian** [comment]
@CS2103AUG2016-W15-C3/integration Ready for integration

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [![Build Status](https://travis-ci.org/CS2103AUG2016-W15-C3/main.svg?branch=master)](https://travis-ci.org/CS2103AUG2016-W15-C3/main)
-[![Coverage Status](https://coveralls.io/repos/github/CS2103AUG2016-W15-C3/main/badge.svg?branch=master)](https://coveralls.io/github/CS2103AUG2016-W15-C3/main?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/CS2103AUG2016-W15-C3/main/badge.svg)](https://coveralls.io/github/CS2103AUG2016-W15-C3/main)
```
Comment (turtle96) [inline README.md]: Should point to master branch

Comment (turtle96) [inline README.md]: Should point to master branch

Comment (marcusngwenjian) [reply README.md]: Resolved

**Pull request merged**